python/lsst/validate/drp/matchreduce.py

```diff
@@ -301,12 +304,13 @@ def _loadAndMatchCatalogs(self, repo, dataIds, matchRadius,
                     tmpCat['base_PsfFlux_mag'][:] = _[0]
                     tmpCat['base_PsfFlux_magErr'][:] = _[1]
 
-            _, psf_e1, psf_e2 = ellipticity_from_cat(oldSrc, slot_shape='slot_PsfShape')
-            _, star_e1, star_e2 = ellipticity_from_cat(oldSrc, slot_shape='slot_Shape')
-            tmpCat['e1'][:] = star_e1
-            tmpCat['e2'][:] = star_e2
-            tmpCat['psf_e1'][:] = psf_e1
-            tmpCat['psf_e2'][:] = psf_e2
+            if not self.skipTEx:
```

Comment:
Oh! Good catch. Since this is an internal method that's called by `__init__`, I'm just going to use the class member and cleanup the method signature.

Reply:
Great!